Userland/Libraries/LibGUI/ComboBox.cpp

```diff
@@ -274,11 +274,21 @@ String ComboBox::text() const
     return m_editor->text();
 }
 
-void ComboBox::set_text(const String& text)
+void ComboBox::set_text(String const& text)
```

Comment:
Please rebase on latest master; with clang-format 14 we get east const for free.